03_algorithm_selection.ipynb

```diff
@@ -17,7 +17,7 @@
    "source": [
```

Comment:
<p>This cell throws error, seems to be an old cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/ml_weather_tutorial/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>It seems to have gotten out of order, it needed to be moved down to after the X_test and y_test were defined.</p>